model/src/main/resources/reference.conf

```diff
@@ -0,0 +1,22 @@
+// A reference configuration file containing all possible ORT configuration options. Some of those options are mutually
+// exclusive, so this file is only used to show all options and to validate the configuration.
+ort {
+  scanner {
+    artifactoryStorage {
+      url = "https://your-artifactory"
+      repository = repository
+      apiToken = apiToken
+    }
+
+    localFileStorage {
+      directory = ~/.ort/scanner
```

Comment:
Note: For the `~` to actually work here we'd need to call `expandTilde()` in line 68 of `ScanResultsStorage.kt` (this is a limitation that was present before your changes).

Reply:
See https://github.com/heremaps/oss-review-toolkit/pull/1847.

Reply:
Thanks.